From: Qingqing <wangyiqing@kylinos.cn>
Date: Fri, 14 Aug 2026 08:14:14 +0000
Subject: Translated using Weblate (Thai)

Currently translated at 100.0% (39 of 39 strings)

Translation: openkylin-huanghe/ukcc-plugin
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/ukcc-plugin/th/
---
 panel/ukcc-plugin/translations/th.ts | 78 ++++++++++++++++++------------------
 1 file changed, 39 insertions(+), 39 deletions(-)

diff --git a/panel/ukcc-plugin/translations/th.ts b/panel/ukcc-plugin/translations/th.ts
index 894b8a5..ab72af8 100644
--- a/panel/ukcc-plugin/translations/th.ts
+++ b/panel/ukcc-plugin/translations/th.ts
@@ -5,168 +5,168 @@
     <name>PanelUkccPlugin</name>
     <message>
         <source>Panel</source>
-        <translation type="unfinished"></translation>
+        <translation>แถบงาน</translation>
         <extra-contents_path>/Panel/Panel</extra-contents_path>
     </message>
     <message>
         <source>Merge icons on the taskbar</source>
-        <translation type="unfinished"></translation>
+        <translation>รวมไอคอนบนแถบงาน</translation>
         <extra-contents_path>/Panel/Merge icons on the taskbar</extra-contents_path>
     </message>
     <message>
         <source>Always</source>
-        <translation type="unfinished"></translation>
+        <translation>เสมอ</translation>
     </message>
     <message>
         <source>Never</source>
-        <translation type="unfinished"></translation>
+        <translation>ไม่เคย</translation>
     </message>
     <message>
         <source>Panel location</source>
-        <translation type="unfinished"></translation>
+        <translation>ตำแหน่งแถบงานบนหน้าจอ</translation>
         <extra-contents_path>/Panel/Panel location</extra-contents_path>
     </message>
     <message>
         <source>Bottom</source>
-        <translation type="unfinished"></translation>
+        <translation>ด้านล่าง</translation>
     </message>
     <message>
         <source>Left</source>
-        <translation type="unfinished"></translation>
+        <translation>ด้านซ้าย</translation>
     </message>
     <message>
         <source>Top</source>
-        <translation type="unfinished"></translation>
+        <translation>ด้านบน</translation>
     </message>
     <message>
         <source>Right</source>
-        <translation type="unfinished"></translation>
+        <translation>ด้านขวา</translation>
     </message>
     <message>
         <source>Panel size</source>
-        <translation type="unfinished"></translation>
+        <translation>ขนาดแถบงาน</translation>
         <extra-contents_path>/Panel/Panel size</extra-contents_path>
     </message>
     <message>
         <source>Small</source>
-        <translation type="unfinished"></translation>
+        <translation>เล็ก</translation>
     </message>
     <message>
         <source>Medium</source>
-        <translation type="unfinished"></translation>
+        <translation>กลาง</translation>
     </message>
     <message>
         <source>Large</source>
-        <translation type="unfinished"></translation>
+        <translation>ใหญ่</translation>
     </message>
     <message>
         <source>Custom</source>
-        <translation type="unfinished"></translation>
+        <translation>กำหนดเอง</translation>
     </message>
     <message>
         <source>Panel auto hide</source>
-        <translation type="unfinished"></translation>
+        <translation>ซ่อนแถบงานอัตโนมัติ</translation>
         <extra-contents_path>/Panel/Panel auto hide</extra-contents_path>
     </message>
     <message>
         <source>Panel lock</source>
-        <translation type="unfinished"></translation>
+        <translation>ล็อกแถบงาน</translation>
         <extra-contents_path>/Panel/Panel lock</extra-contents_path>
     </message>
     <message>
         <source>Widgets always showed on panel</source>
-        <translation type="unfinished"></translation>
+        <translation>ไอคอนที่แสดงบนแถบงานเสมอ</translation>
         <extra-contents_path>/Panel/Widgets always showed on panel</extra-contents_path>
     </message>
     <message>
         <source>Task view</source>
-        <translation type="unfinished"></translation>
+        <translation>มุมมองงานหลายงาน</translation>
     </message>
     <message>
         <source>Volume</source>
-        <translation type="unfinished"></translation>
+        <translation>เสียง</translation>
     </message>
     <message>
         <source>Network</source>
-        <translation type="unfinished"></translation>
+        <translation>เครือข่าย</translation>
     </message>
     <message>
         <source>Power</source>
-        <translation type="unfinished"></translation>
+        <translation>พลังงาน</translation>
     </message>
     <message>
         <source>Bluetooth</source>
-        <translation type="unfinished"></translation>
+        <translation>บลูทูธ</translation>
     </message>
     <message>
         <source>Icons showed on system tray</source>
-        <translation type="unfinished"></translation>
+        <translation>แสดงไอคอนในถาดระบบ</translation>
         <extra-contents_path>/Panel/Icons showed on system tray</extra-contents_path>
     </message>
     <message>
         <source>Search</source>
-        <translation type="unfinished"></translation>
+        <translation>ค้นหา</translation>
     </message>
     <message>
         <source>When existing multiple panels, show window icons on</source>
-        <translation type="unfinished"></translation>
+        <translation>เมื่อมีแถบงานหลายแถบ ไอคอนจะแสดงบน</translation>
     </message>
     <message>
         <source>All panels</source>
-        <translation type="unfinished"></translation>
+        <translation>แถบงานทั้งหมด</translation>
     </message>
     <message>
         <source>Primary screen panel and panel where window is open</source>
-        <translation type="unfinished"></translation>
+        <translation>แถบงานของหน้าจอหลักและแถบงานที่เปิดหน้าต่างอยู่</translation>
     </message>
     <message>
         <source>Panel where window is open</source>
-        <translation type="unfinished"></translation>
+        <translation>แสดงเฉพาะแถบงานที่เปิดหน้าต่างอยู่เท่านั้น</translation>
     </message>
     <message>
         <source>Panel Appearance</source>
-        <translation type="unfinished"></translation>
+        <translation>ลักษณะแถบงาน</translation>
     </message>
     <message>
         <source>classic</source>
-        <translation type="unfinished"></translation>
+        <translation>คลาสสิก</translation>
     </message>
     <message>
         <source>threeIsland</source>
-        <translation type="unfinished"></translation>
+        <translation>มิชิมะ</translation>
     </message>
     <message>
         <source>Show panel on all screens</source>
-        <translation type="unfinished"></translation>
+        <translation>แสดงแถบงานบนหน้าจอทั้งหมด</translation>
         <extra-contents_path>/Panel/Show panel on all screens</extra-contents_path>
     </message>
     <message>
         <source>Show system tray area on all panels </source>
-        <translation type="unfinished"></translation>
+        <translation>แสดงพื้นที่การดำเนินการของถาดบนแถบงานทั้งหมด </translation>
     </message>
     <message>
         <source>Virtual Keyboard</source>
-        <translation type="unfinished"></translation>
+        <translation>แป้นพิมพ์เสมือน</translation>
     </message>
     <message>
         <source>VPN</source>
-        <translation type="unfinished"></translation>
+        <translation>เครื่องมือ VPN</translation>
     </message>
     <message>
         <source>Fcitx</source>
-        <translation type="unfinished"></translation>
+        <translation>วิธีการป้อนข้อมูล</translation>
     </message>
     <message>
         <source>Never Show</source>
-        <translation type="unfinished"></translation>
+        <translation>ไม่แสดงเลย</translation>
     </message>
     <message>
         <source>Always Show</source>
-        <translation type="unfinished"></translation>
+        <translation>แสดงเสมอ</translation>
     </message>
     <message>
         <source>Show When Keyboard Is Connected</source>
-        <translation type="unfinished"></translation>
+        <translation>แสดงเมื่อไม่ได้เชื่อมต่อแป้นพิมพ์</translation>
     </message>
 </context>
 <context>
